src/environments/.env.ts

```
@@ -1,9 +1,17 @@
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
 // IMPORTANT: THIS FILE IS AUTO GENERATED! DO NOT MANUALLY EDIT OR CHECKIN!
 /* tslint:disable */
 export default {
   'mifos_x': {
-    'version': '260108',
-    'hash': 'g5af11fe75'
+    'version': '260428',
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The 'version': '260428', key has incorrect delimiter

(IncorrectDelimiter)